extensions/ffmpeg/src/main/java/com/google/android/exoplayer2/ext/ffmpeg/FfmpegDecoder.java

```diff
@@ -87,7 +87,11 @@ public FfmpegDecoderException decode(DecoderInputBuffer inputBuffer,
     }
     if (!hasOutputFormat) {
       channelCount = ffmpegGetChannelCount(nativeContext);
-      sampleRate = ffmpegGetSampleRate(nativeContext);
+      if ("alac".equals(codecName)) {
```

Comment:
I did some digging and couldn't find anything that indicated that it was intentionally not being set. It may be a bug since the behavior is inconsistent with the other decoders. I can submit a ticket to FFmpeg if needed.